test/common/stats/symbol_table_impl_test.cc

```diff
@@ -142,21 +142,30 @@ TEST_P(StatNameTest, TestEmpty) {
 }
 
 TEST_P(StatNameTest, TestDynamic100k) {
-  // Tests 100k different sizes of dynamic stat, covering all kinds of
-  // corner cases of spilling over into multi-byte lengths.
+  // Tests a variety different sizes of dynamic stat ranging from 3 to 256k=1,
```

Comment:
256k-1? Not `=`

Reply:
Also, after looking at the code and the comment for about 2 minutes, I can't figure out where either 3 or 256k are represented in the code.